src/main.c

```diff
@@ -337,12 +340,34 @@ static void init_windows(void)
 	wprintw(infobox, _(" r - Redraw\n"));
 	wprintw(infobox, _(" S - Solve puzzle\n"));
 	wprintw(infobox, _(" x - Delete number\n"));
+	wprintw(infobox, _(" u - Undo previous move\n"));
```

Comment:
We are not undoing moving in the grid but undoing insertions/actions

Reply:
Yeah I was thinking move in the game sense like a chess "move", not a traversing the board "move." I like action as it covers both insertion and deletion